lib/token-manager/token-manager.js

```diff
@@ -22,6 +22,7 @@ const PublicKeyUtil = require("../utils/public-key-util");
 
 const TOKEN_PATH = '/token';
 const CUSTOM_IDENTITY_GRANT_TYPE = 'urn:ietf:params:oauth:grant-type:jwt-bearer';
+const APP_TO_APP_GRANT_TYPE = 'client_credentials';
```

Comment:
can this constant be defined in 
https://github.com/ibm-cloud-security/appid-serversdk-nodejs/blob/master/lib/utils/constants.js ?